tex/measure/large-laws.tex

```diff
@@ -60,6 +60,272 @@ \subsection{Convergence in probability}
 
 \subsection{Convergence in law}
 
+\section{Weak law of large numbers}
+
+As the name implies, this is a direct corollary of the strong law of large numbers.
+Nevertheless, the proof of this law is simpler, and some applications only require the weak law.
+
+\todo{write}
+
+\subsection{Application: Weierstrass approximation}
+
+\section{Strong law of large numbers}
+
+\subsection{Motivation: Biased random walk}
+
+Consider a random walk defined as follows:
+\begin{itemize}
+	\ii Let $X_0 = 1$.
+	\ii For each $i \geq 1$, define $X_i$ to be $X_{i-1}-1$ with probability $p=0.6$
+	or $X_{i-1}+1$ with probability $1-p=0.4$.
+\end{itemize}
+
+Then we can ask: What's the expected number of steps until some $X_i$ equals $0$?
+
+A naive attempt might be the following.
+\begin{quote}
+	Let $f(i)$ be the expected number of steps starting to reach $0$ starting from $X_0=i$.
+
+	Then:
+	\begin{itemize}
+		\ii $f(0)=0$,
+		\ii $f(1)=1+0.6 f(0)+0.4 f(2)$,
+		\ii $f(2)=1+0.6 f(1)+0.4 f(3)$,
+		\ii $\vdots$
+	\end{itemize}
+\end{quote}
+This isn't getting anywhere because there are infinitely many terms.
+A better attempt is the following:
+\begin{quote}
+	Let the answer be $x$.
+	If we start from $X_0=2$, let $i$ be the first time such that $X_i=1$
+	and $j$ be the first time after $i$ such that $X_j=0$. Then
+	\[ \EE[i]=\EE[j-i]=x. \]
+	Therefore,
+	\[ x = 1 + 0.6 \cdot 0 + 0.4 \cdot (2x) \]
+	Solving the equation, we get $x=5$.
+\end{quote}
+It gives the correct result --- however, the same method gives $x=-5$ when
+the probability of going down is $p=0.4$, which is clearly nonsense.
+
+What went wrong? The problem is when $p=0.4$, there is a nonzero probability\footnote{%
+	Preview: Using martingale theory next chapter, you will be able to prove the probability
+	the sequence never reaches $0$ is exactly $1-\frac{0.4}{0.6}$.
+} that the
+sequence never reaches $0$, so the expected value is undefined
+and we're subtracting $\infty$ from $\infty$ in the proof.
+
+In this case, the strong law of large numbers can help us patch this hole, by showing that in almost
+every world, the sequence $X_i$ eventually reaches $0$.
```

Comment:
I read about this application somewhere (Tony's notes???) but I forgot the source